design-documents/graph-ql/coverage/negotiableQuotes.graphqls

```diff
@@ -0,0 +1,147 @@
+type Mutation {
+    createNegotiableQuote(
+        cart_id: ID! @doc(description: "Cart ID")
+        quote_name: String @doc(description: "Quote name")
+        comment: String @doc(description: "Comment")
+        files: [FileInput!] @doc(description: "Attached files")
+    ): RequestNegotiableQuoteOutput! @doc(description:"Generates negotiable quote request")
+
+    setShippingMethodsOnNegotiableQuote(
+        cart_id: ID! @doc(description: "Cart ID")
+        shipping_methods: [String!]! @doc(description: "Input for shipping method")
+    ): SetShippingMethodsOnNegotiableQuoteOutput @doc(description:"Set negotiable quote shipping method")
+}
+
+type Customer {
+    negotiable_quote (
+        id: ID!): NegotiableQuote @doc(description: "Get negotiable quote of a customer")
+
+    negotiable_quotes(
+        page_size: Int = 20,
+        current_page: Int = 1,
+        filter: NegotiableQuoteFilterInput
+    ): [NegotiableQuote] @doc(description: "Get negotiable quotes of a customer")
+
+    estimateShippingCostForNegotiableQuote(
```

Comment:
This query is excessive and its functionality covered by `Customer => negotiable_quote => available_shipping_methods`
```suggestion
```